#! /usr/bin/env python

PACKAGE='hanse_wall_sonar'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
#       Name				Type		Level	Description				Default	Min	Max
gen.add("simulation_mode_",		bool_t,		0,	"en-/disables the simulation mode",	False)
gen.add("limit_lookahead_distance_",	double_t,	0,	"distance to farest point",		10,	0,	50)
gen.add("limit_lookahead_index_delta_",int_t,		0,	"max number of points",			20,	1,	50)
gen.add("wall_distance_right_", double_t, 0, "distance to the wall on the right side", -1.5, -20,20)

exit(gen.generate(PACKAGE, "wall_follow_shift_node", "wall_follow_shift_node_params"))
